src/app/sidebar/Sidebar.tsx

```diff
@@ -19,6 +18,9 @@ export interface SidebarProps {
 
   selectedCourse?: SelectedCourse;
   setSelectedCourse: Dispatch<SetStateAction<SelectedCourse | undefined>>;
+
+  selectedSubject?: string;
+  setSelectedSubject: Dispatch<SetStateAction<string | undefined>>;
```

Comment:
maybe we should use callbacks instead of `Dispatch` now?

Reply:
@keithradford check out the changes :+1: